integrationtests/src/test/java/org/axonframework/integrationtests/polymorphic/AbstractPolymorphicAggregateAnnotationCommandHandlerTestSuite.java

```diff
@@ -67,28 +69,30 @@ void setUp() {
 
         transactionManager = new EntityManagerTransactionManager(entityManager);
 
-        CommandBus commandBus = SimpleCommandBus.builder()
-                                                .transactionManager(transactionManager)
-                                                .build();
+        commandBus = SimpleCommandBus.builder()
+                                     .transactionManager(transactionManager)
+                                     .build();
         commandGateway = DefaultCommandGateway.builder()
                                               .commandBus(commandBus)
                                               .build();
 
         AggregateModel<ParentAggregate> model = new AnnotatedAggregateMetaModelFactory()
-                .createModel(ParentAggregate.class, new HashSet<>(asList(Child1Aggregate.class, Child2Aggregate.class)));
+                .createModel(ParentAggregate.class,
+                             new HashSet<>(asList(Child1Aggregate.class, Child2Aggregate.class)));
```

Comment:
Does subTypes strictly need to be a Set? The API would be friendlier is it's just a Collection or Iterable.
Or perhaps have the static method defined as:
```
public static <T> AggregateModel<T> inspectAggregate(Class<T> aggregateType, Class<? extends T>... subtypes) 
```
Inside that static method, you could add them to a set to have that set used throughout the rest of the inspector.

Reply:
Not overly certain on this part, but I recall suggesting the usages of `Set` earlier in the PR. Reasoning behind that would've been to ensure no duplicate aggregate sub-types would be inserted. If that advantage is outweighed by using an `Iterable`, I'll back-off.

Reply:
I have the same concern here about clarity if we choose to go for varargs.

Reply:
@abuijze and I decided that approach with `(Class<T>, Class<? extends T>...)` looks pretty weird for usage, so we won't be pursuing it.